config/webpack.dev.js

```diff
@@ -1,23 +1,33 @@
 const merge = require('webpack-merge');
 const webpack = require('webpack');
 const config = require('./webpack.base');
-const GLOBALS = {
-    'process.env': {
-        NODE_ENV: JSON.stringify('development'),
-    },
-};
 
 module.exports = merge.smart(config, {
+    mode: 'development',
+    devtool: 'cheap-module-source-map',
     cache: true,
-    devtool: 'inline-source-map',
     entry: {
         client: ['webpack-hot-middleware/client', 'client'],
     },
     plugins: [
-        new webpack.DefinePlugin(GLOBALS),
+        new webpack.DefinePlugin({
+            'process.env.NODE_ENV': JSON.stringify(
+                process.env.NODE_ENV || 'development'
+            ),
+        }),
         new webpack.HotModuleReplacementPlugin(),
     ],
     module: {
         rules: [],
     },
+    node: {
```

Comment:
Why do we need to create these empty objects? Are there things running in browser that expect node globals?

Reply:
I used to have it because some libraries import Node modules but don't use them in the browser. Maybe we don't need it in this boiler, I'll remove it.